NOM_visualizations/R/NOM_R_notebook.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>Can you remove the groupColor from ggMarginal call to prevent warning messages?  </p><p> </p><p>Two small suggested improvements on this awesome plot.</p><p>1) Make background bw (maybe with <code>theme_bw()</code>?)</p><p>2) Make markers solid (shape = 19 in your geom_point)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>done!</p>